packages/app-layout/src/vaadin-app-layout.js

```diff
@@ -260,10 +261,7 @@ class AppLayout extends ElementMixin(ThemableMixin(ControllerMixin(PolymerElemen
         @media (max-width: 800px), (max-height: 600px) {
           :host {
             --vaadin-app-layout-drawer-overlay: true;
-          }
-
-          [part='drawer'] {
-            width: 20em;
+            --vaadin-app-layout-drawer-width: 20em;
```

Comment:
If you set:
```css
vaadin-app-layout {
  --vaadin-app-layout-drawer-width: 300px;
}
```

...it will be stronger than this value and the drawer width will have the same width on wide and narrow screen. Not sure what is the desired outcome. Should there be different properties for the two modes or should the default here be calculated from whatever the value is on the host by default?

Reply:
> Should there be different properties for the two modes

I don’t think so. Should be enough to scope them based on the mode attribute:

```css
vaadin-app-layout {
  --vaadin-app-layout-drawer-width: 100px;
}

vaadin-app-layout[overlay] {
  -vaadin-app-layout-drawer-width: 200px;
}
```